notebooks/text_models/solutions/text_generation_using_transformers.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>In <code>dir = os.path.expanduser</code> maybe use a different variable name (e.g. <code>data_path</code> )  than <code>dir</code>, since <code>dir</code> is a reserved keyword.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Good catch! Updated.</p>